Doc/source/coordinates.rst

```diff
@@ -1,32 +1,99 @@
+#############################################################################################
+spacepy.coordinates - Implementation of Coords class functions for coordinate transformations
```

Comment:
Let's not include the "spacepy" in these titles, since they will be going into indices and tables of contents and such.

Reply:
Also an easy fix, this one is resolved